docs/modules/ROOT/pages/api/erc721.adoc

```diff
@@ -160,19 +160,19 @@ Interface for the optional metadata functions in {eip721}.
 
 [.contract-item]
 [[IERC721Metadata-name]]
-==== `[.contract-item-name]#++name++#++() -> felt252++` [.item-kind]#external#
+==== `[.contract-item-name]#++name++#++() -> ByteArray++` [.item-kind]#external#
 
 Returns the NFT name.
 
 [.contract-item]
 [[IERC721Metadata-symbol]]
-==== `[.contract-item-name]#++symbol++#++() -> felt252++` [.item-kind]#external#
+==== `[.contract-item-name]#++symbol++#++() -> ByteArray++` [.item-kind]#external#
 
 Returns the NFT ticker symbol.
 
 [.contract-item]
 [[IERC721Metadata-token_uri]]
-==== `[.contract-item-name]#++token_uri++#++(token_id: u256) -> felt252++` [.item-kind]#external#
+==== `[.contract-item-name]#++token_uri++#++(token_id: u256) -> ByteArray++` [.item-kind]#external#
 
 Returns the Uniform Resource Identifier (URI) as a short string for the `token_id` token.
 If the URI is not set for `token_id`, the return value will be `0`.
```

Comment:
Is 0 the default value for ByteArrays?

Reply:
Nope, should be an empty ByteArray. Fixed!